sieve_of_eratosthenes/sieve_of_eratosthenes.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+// N = Limit, change N to get all primes <= N
+const N = 20
+
+// SieveOfEratosthenes returns bool array primes
+func SieveOfEratosthenes() [N + 1]bool {
```

Comment:
You are taking N as a const. But suppose someone wants to use your code in a program. Don't you think making `n` a normal function variable will be better?